dep_updates.md

```diff
@@ -0,0 +1,55 @@
+# Sinopia Weekly Dependency Updates
+
+This is meant to be a guide to DLSS Infrastructure Team on how to do the weekly dependency update work for all the Sinopia related bits we maintain.
+
+## Which Repos?
+
+The repos that get weekly dependency updates:
+
+(from https://github.com/sul-dlss/access-update-scripts/blob/master/infrastructure/projects)
+- LD4P/sinopia_api
+- LD4P/sinopia_editor
+- LD4P/sinopia_exporter
+- LD4P/sinopia_indexing_pipeline
+
+## For Deployed Development Environment
+
+After the automated dependency update PRs are merged (via script, usually - see [First Responder doc](https://github.com/sul-dlss/DeveloperPlaybook/blob/master/infrateam_first_responder.md#merge-em)), CircleCI will automatically build a docker image and push it to DockerHub.
+
+(See https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/sinopia/operations-concerns.md#development)
+
+## For Stage and Production Environments
+
+You do NOT need to create a github tagged release *issue* such as https://github.com/LD4P/sinopia_editor/issues/2755 for dependency updates.
```

Comment:
What is a "tagged release issue"?  Should we get rid of that checklist then?

Reply:
Yes, I don't understand why this wasn't just added to the checklist. This seems redundant.

Reply:
there is a github template for a "tagged release" that has a bunch of steps for the product owner to weigh in and so on.  It's really for when we're in a Sinopia work cycle and we want to do a minor release, not a bug fix release.  See the example indicated https://github.com/LD4P/sinopia_editor/issues/2755 and note that 1) it hung around open for a long time 2) michelle had no interest in weighing in   3) there were a lot of irrelevant steps.    

I wrote this document because I felt between the FR doc, the devopsdocs, the sinopia_editor README and any place else I was directed to, there was information about how to do ... not exactly what we do every week.  And I didn't like being asked to look in 3+ different places, only to have to figure out the right process for weekly dep updates on my own.

I don't keep Sinopia process in my head -- it falls out.  So I need the recipe for the simple minded -- so I wrote it.

Reply:
I follow the "tagged release" template every time I'm responsible for any release (major, minor, bug, dependency update) and just ignore the step for the product owner when not necessary.